src/appleseed.cli/stdouttilecallback.cpp

```diff
@@ -141,7 +141,11 @@ namespace
             // This header is sent only once and can contains AOVs and frame informations.
             const bool beauty_only = (m_export_options == StdOutTileCallbackFactory::TileOutputOptions::BeautyOnly);
             const size_t chunk_size = 1 * sizeof(uint32);
-            const size_t plane_count = beauty_only ? 1 : 1 + frame.aovs().size();
+            const size_t plane_count =
+                beauty_only ? 1 : 1
+                + frame.aovs().size()
+                + (frame.has_extra_aov() ? frame.extra_aov_indexs().size() : 0);
```

Comment:
Why not just `+ frame.extra_aov_indexs().size()` (which can be 0 if there is none)?

Reply:
I prefer that we find an alternative way of doing it. Maybe converting the diagnostic AOVs into normal AOVs or something similar.

Also, note that this won't work in the new blenderseed addon. 
The new addon does not use stdout to pass image data to Blender.

Reply:
Note that `--to-stdout` is not exclusive to blenderseed.